packages/zoltan2/test/core/color/CMakeLists.txt

```diff
@@ -30,13 +30,487 @@ TRIBITS_ADD_TEST(
+  NAME Coloring1_complete_D2
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=complete --colorMethod=D2"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_simple_PD2
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=simple --colorMethod=PD2"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_simple_PD2_recolor_degrees
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+  "--inputFile=simple --colorMethod=PD2 --recolorDegrees"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_simple_PD2_serial_threshold_25
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=simple --colorMethod=PD2 --serialThreshold=25"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+IF (${PACKAGE_NAME}_ENABLE_ParMETIS)
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_simple_PD2_rows
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=simple --colorMethod=PD2 --prepartition=rows"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_simple_PD2_nonzeros
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=simple --colorMethod=PD2 --prepartition=nonzeros"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+ENDIF()
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_commanchedual_PD2
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=commanche_dual --colorMethod=PD2"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+IF (${PACKAGE_NAME}_ENABLE_ParMETIS)
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_commanchedual_PD2_rows
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=commanche_dual --colorMethod=PD2 --prepartition=rows"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_commanchedual_PD2_nonzeros
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=commanche_dual --colorMethod=PD2 --prepartition=nonzeros"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+ENDIF()
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_noedges_PD2
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=noedges --colorMethod=PD2"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
+
+TRIBITS_ADD_TEST(
+  Coloring1
+  NAME Coloring1_complete_PD2
+  NUM_MPI_PROCS 4
+  COMM serial mpi
+  ARGS
+   "--inputFile=complete --colorMethod=PD2"
+  PASS_REGULAR_EXPRESSION "PASS"
+  FAIL_REGULAR_EXPRESSION "FAIL"
+  )
 ENDIF()
 
+
 # Command that copies files to the executable directory.
 TRIBITS_COPY_FILES_TO_BINARY_DIR(copy_files_for_color_tests
   SOURCE_FILES
     simple.mtx
     commanche_dual.mtx
+    complete.mtx
```

Comment:
This feedback was hiding in a collapsed menu. I could come up with a small input pretty easily. I am not sure it would add to test coverage, so this is low on my priorities right now.